test/libsolidity/util/TestFileParser.cpp

```diff
@@ -602,7 +647,7 @@ string TestFileParser::Scanner::scanIdentifierOrKeyword()
 {
 	string identifier;
 	identifier += current();
-	while (langutil::isIdentifierPart(peek()))
+	while (langutil::isIdentifierPartWithDot(peek()))
```

Comment:
Since this is not used elsewhere (the scanner creates identifier + period), perhaps it is better to just check manually here (to avoid confusion)?
```
while (languitl::isIdentifier(peek()) || peek() == '.')
```

Or rename it to `isPath`?

Reply:
I would prefer the first proposal. Path can also contain whitespace.

Reply:
Oh, this changes how any identifier is scanned in the tests? Do we really want that? Maybe we can also just use `_` as a separator since there is no real hierarchy anymore?

Reply:
Ok, lets use `_`.